pkg/csi/recover/recover.go

```diff
@@ -41,10 +41,12 @@ import (
 )
 
 const (
-	defaultKubeletTimeout     = 10
-	defaultFuseRecoveryPeriod = 5 * time.Second
-	serviceAccountTokenFile   = "/var/run/secrets/kubernetes.io/serviceaccount/token"
-	FuseRecoveryPeriod        = "RECOVER_FUSE_PERIOD"
+	defaultKubeletTimeout          = 10
+	defaultFuseRecoveryPeriod      = 5 * time.Second
+	defaultRecoverWarningThreshold = 50
+	serviceAccountTokenFile        = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+	FuseRecoveryPeriod             = "RECOVER_FUSE_PERIOD"
+	RecoverWarningThreshold        = "REVOCER_WARNING_THRESHOLD"
```

Comment:
mistake, REVOCER_WARNING_THRESHOLD  -> RECOVER_WARNING_THRESHOLD

Reply:
Thanks for figuring it out! Would you mind to create a PR to fix it?